### ISSUE TITLE
issues with ```org.apache.tika:tika-app:1.13```

### ISSUE THREAD

**Issue by volth** [coursier/coursier#586]
```coursier resolve org.apache.tika:tika-app:1.13```
results in
```
c3p0:c3p0:0.9.1.1
com.adobe.xmp:xmpcore:5.1.2
com.beust:jcommander:1.35
com.drewnoakes:metadata-extractor:2.8.1
com.fasterxml.jackson.core:jackson-core:2.7.1
com.github.junrar:junrar:0.7
com.github.virtuald:curvesapi:1.03
com.google.code.gson:gson:2.6.2
com.google.guava:guava:17.0
com.google.protobuf:protobuf-java:2.5.0
com.googlecode.json-simple:json-simple:1.1.1
com.googlecode.juniversalchardet:juniversalchardet:1.0.3
com.googlecode.mp4parser:isoparser:1.1.18
com.healthmarketscience.jackcess:jackcess:2.1.3
com.healthmarketscience.jackcess:jackcess-encrypt:2.1.1
com.intellij:annotations:12.0
com.optimaize.languagedetector:language-detector:0.5
com.pff:java-libpst:0.8.1
com.rometools:rome:1.5.1
com.rometools:rome-utils:1.5.1
commons-cli:commons-cli:1.3.1
commons-codec:commons-codec:1.10
commons-io:commons-io:2.4
commons-lang:commons-lang:2.6
commons-logging:commons-logging:1.2
commons-logging:commons-logging-api:1.1
de.l3s.boilerpipe:boilerpipe:1.1.0
edu.ucar:cdm:4.5.5
edu.ucar:grib:4.5.5
edu.ucar:httpservices:4.5.5
edu.ucar:jj2000:5.2
edu.ucar:netcdf4:4.5.5
edu.ucar:udunits:4.5.5
javax.annotation:javax.annotation-api:1.2
javax.measure:jsr-275:0.9.3
javax.ws.rs:javax.ws.rs-api:2.0.1
joda-time:joda-time:2.2
log4j:log4j:1.2.17
net.arnx:jsonic:1.2.11
net.java.dev.jna:jna:4.1.0
net.jcip:jcip-annotations:1.0
net.sf.ehcache:ehcache-core:2.6.2
net.sf.jwordnet:jwnl:1.3.3
net.sourceforge.jmatio:jmatio:1.0
org.apache.commons:commons-compress:1.11
org.apache.commons:commons-csv:1.0
org.apache.commons:commons-exec:1.3
org.apache.commons:commons-vfs2:2.0
org.apache.cxf:cxf-core:3.0.3
org.apache.cxf:cxf-rt-frontend-jaxrs:3.0.3
org.apache.cxf:cxf-rt-rs-client:3.0.3
org.apache.cxf:cxf-rt-transports-http:3.0.3
org.apache.httpcomponents:httpclient:4.2.6
org.apache.httpcomponents:httpcore:4.2.5
org.apache.httpcomponents:httpmime:4.2.6
org.apache.james:apache-mime4j-core:0.7.2
org.apache.james:apache-mime4j-dom:0.7.2
org.apache.maven.scm:maven-scm-api:1.4
org.apache.maven.scm:maven-scm-provider-svn-commons:1.4
org.apache.maven.scm:maven-scm-provider-svnexe:1.4
org.apache.opennlp:opennlp-maxent:3.0.3
org.apache.opennlp:opennlp-tools:1.5.3
org.apache.pdfbox:fontbox:2.0.1
org.apache.pdfbox:jempbox:1.8.12
org.apache.pdfbox:pdfbox:2.0.1
org.apache.pdfbox:pdfbox-debugger:2.0.1
org.apache.pdfbox:pdfbox-tools:2.0.1
org.apache.poi:poi:3.15-beta1
org.apache.poi:poi-ooxml:3.15-beta1
org.apache.poi:poi-ooxml-schemas:3.15-beta1
org.apache.poi:poi-scratchpad:3.15-beta1
org.apache.sis.core:sis-metadata:0.6
org.apache.sis.core:sis-referencing:0.6
org.apache.sis.core:sis-utility:0.6
org.apache.sis.storage:sis-netcdf:0.6
org.apache.sis.storage:sis-storage:0.6
org.apache.tika:tika-app:1.13
org.apache.tika:tika-batch:1.13
org.apache.tika:tika-core:1.13
org.apache.tika:tika-langdetect:1.13
org.apache.tika:tika-parsers:1.13
org.apache.tika:tika-serialization:1.13
org.apache.tika:tika-xmp:1.13
org.apache.ws.xmlschema:xmlschema-core:2.1.0
org.apache.xmlbeans:xmlbeans:2.6.0
org.bouncycastle:bcmail-jdk15on:1.54
org.bouncycastle:bcpkix-jdk15on:1.54
org.bouncycastle:bcprov-jdk15on:1.54
org.bouncycastle:bcprov-jdk15on:1.54
org.ccil.cowan.tagsoup:tagsoup:1.2.1
org.codehaus.plexus:plexus-utils:1.5.6
org.codehaus.woodstox:stax2-api:3.1.4
org.codehaus.woodstox:woodstox-core-asl:4.4.1
org.codelibs:jhighlight:1.0.2
org.gagravarr:vorbis-java-core:0.8
org.gagravarr:vorbis-java-tika:0.8
org.itadaki:bzip2:0.9.1
org.jdom:jdom2:2.0.4
org.json:json:20140107
org.jsoup:jsoup:1.7.2
org.opengis:geoapi:3.0.0
org.ow2.asm:asm:5.0.4
org.quartz-scheduler:quartz:2.2.0
org.slf4j:jcl-over-slf4j:1.7.12
org.slf4j:jul-to-slf4j:1.7.12
org.slf4j:slf4j-api:1.7.12
org.slf4j:slf4j-jdk14:1.7.7
org.slf4j:slf4j-log4j12:1.7.12
org.tukaani:xz:1.5
regexp:regexp:1.3
```

Note that 
1. ```org.bouncycastle:bcprov-jdk15on:1.54``` is duplicated
2. ```org.slf4j:slf4j-jdk14:1.7.7``` shouldn't be here at all (it is a ```test``` scope dependency and also its version does not match the rest of ```slf4j``` jars)

**Comment by alexarchambault** [comment]
Thanks for the report, https://github.com/coursier/coursier/pull/587 should fix those. Well catched! (2. in particular)

**Comment by alexarchambault** [comment]
This should be fixed in `1.0.0-RC5`.
